secure-mobile-push/smp-android-client/securemobilepush/src/main/java/de/adorsys/android/securemobilepush/sms/SmsTool.java

```diff
@@ -21,4 +21,12 @@ public static void requestSMSPermission(@NonNull Activity activity) {
                     REQUEST_CODE_ASK_PERMISSIONS);
         }
     }
+
+    public static void createSmsConfig(@NonNull String BEGIN_INDEX,
```

Comment:
Fixed in commit:https://github.com/adorsys/secure-key-storage/pull/44/commits/eb220e2d2982822d17e2173262b0ad7e0fb3e81c